example/w-nginx/README.md

```diff
@@ -0,0 +1,91 @@
+# `w-nginx`
+
+<br>
+
+This example shows how to use [nginx](https://nginx.org/en/) as a
+reverse proxy together with Dream. To simplify setup, the example runs
+both nginx and Dream in
+[Docker](https://en.wikipedia.org/wiki/Docker_(software)) containers
+with [Docker Compose](https://docs.docker.com/compose/).
+
+<br>
+
+There are several reasons to pair Dream with a reverse proxy. In this
+example, we allow nginx to handle requests for static assets before
+they reach the application server.
+
+<br>
+
+```ocaml
+let body =
+  <html>
+  <body>
+    <h1>Greetings from the Dream Server!</h1>
+    <image src="/static/ocaml.png" alt="The OCaml logo.">
+    </body>
+  </html>
+
+
+let () =
+  Dream.run ~interface:"0.0.0.0"
+  @@ Dream.logger
+  @@ Dream.router [
+    Dream.get "/" (fun _request -> Dream.html body)
+  ]
+  @@ Dream.not_found
+```
```

Comment:
I recommend also linking and/or showing parts of the nginx configuration, and linking to nginx docs rather than its home page.